corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -375,6 +375,8 @@
                     'analytix/js/hubspot',
                     'analytix/js/kissmetrix',
                     'hqwebapp/js/mobile_experience_warning',
+                    'ace-builds/src-min-noconflict/ace',
```

Comment:
added ace in base dependencies because [here](https://github.com/ajaxorg/ace-builds/blob/master/src-min-noconflict/ext-searchbox.js) ace is accessed outside of define function.

Reply:
That's not what this dependency list is for - this is for modules that are used on all (or virtually all) pages. Dealing with modules that use global vars is generally done using [shim config](https://requirejs.org/docs/api.html#config-shim) which we define in [requirejs_config.js](https://github.com/dimagi/commcare-hq/blob/ddeb3dff35c84e1b2bf5e8547cef0423f050a7d0/corehq/apps/hqwebapp/static/hqwebapp/js/requirejs_config.js#L14). That shim config already has an entry for ace - what was happening that wasn't working?